tutorcairn/templates/cairn/apps/clickhouse/migrations.d/0010_modify_graded_bool.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE course_blocks
+MODIFY COLUMN graded Boolean DEFAULT false;
```

Comment:
have u tried running this migrations? 
means while we are adding data to this table in importcoursedata.py, we are saving "true" or "false" in this field which is string not Boolean.
Is it working fine?

Reply:
Yes, good point. I updated the importcoursedata.py script.

Reply:
`"true" if child.graded else "false",` in importcoursedata.py script.
we can simply use `child.graded` now. What do you think?

Reply:
Hmmm I don't think that would work, because `child.graded` would be converted to "True" or "False" (with an uppercase).

Reply:
Okay then we can merge this PR.